### PR TITLE
haxe.xml documentation

### PR DESCRIPTION
**Pull request by markknol**
`haxe.xml.Fast` and `haxe.xml.Printer` documentation.

For those who still have to use xml.

### CODE REVIEW

File: std/haxe/xml/Fast.hx [hunk 1]
```diff
@@ -106,17 +106,106 @@ private class NodeListAccess implements Dynamic<List<Fast>> {
...
+	**/
 	public var innerHTML(get,null) : String;
+
+	/**
+		Access to the first sub element with the given name. 
+		
+		An exception is thrown if the element doesn't exists.
+		Use `hasNode` to check the existence of a node.
+		
+		```
```
Comment (Gama11) [inline std/haxe/xml/Fast.hx]: Missing "haxe"?

Comment (markknol) [inline std/haxe/xml/Fast.hx]: If you mean `haxe.Xml` then no :)

Comment (Gama11) [inline std/haxe/xml/Fast.hx]: I mean after the triple backticks. It seems that one of the code blocks you added has `haxe` there, and the others don't.

Comment (markknol) [inline std/haxe/xml/Fast.hx]: Good point! Its hard to see where the line notesh are  placed btw.

File: std/haxe/xml/Fast.hx [hunk 2]
```diff
@@ -106,17 +106,106 @@ private class NodeListAccess implements Dynamic<List<Fast>> {
...
+		
+		An exception is thrown if there is no data or if there not only data
+		but also other nodes.
+	**/
 	public var innerData(get,null) : String;
+
+	/**
+		The XML string built with all the sub nodes, excluding the current one.
+	**/
 	public var innerHTML(get,null) : String;
```
Comment (nadako) [inline std/haxe/xml/Fast.hx]: It's funny that it's called `innerHTML` when we're dealing with XML :)

**Comment by nadako** [comment]
Thanks!

**Pull request merged**